src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_editor.cljs

```diff
@@ -168,7 +171,17 @@
   {:get-initial-state
    (fn [{:keys [props]}]
      {:config (:config props)
-      :editing? false})
+      :editing? false
+      :sidebar-visible? true})
    :render
    (fn [{:keys [state refs]}]
-     (render-display state refs))})
+     (render-display state refs))
+   :component-did-mount
+   (fn [{:keys [state refs]}]
+     (set! js/window.onscroll
```

Comment:
Good question! It's as simple as applying this totally godawful pattern:
```
:component-did-mount
(fn [{:keys [this]}]
  (set! (.-onScrollHandler this)
    (fn [] (let [visible...
  (.addEventListener "scroll" (.-onScrollHandler this)))
:component-will-unmount
(fn [{:keys [this]}]
  (.removeEventListener "scroll" (.-onScrollHandler this)))
```
I am so, so sorry.